src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/repository/CourtCaseRepository.java

```diff
@@ -35,12 +32,14 @@
             nativeQuery=true)
     List<CourtCaseEntity> findByCourtCodeAndSessionStartTime(String courtCode, LocalDateTime sessionStartAfter, LocalDateTime sessionStartBefore, LocalDateTime createdAfter);
 
-    @Query(value = "Select cc FROM CourtCaseEntity cc "
-        + "WHERE cc.courtCode = :courtCode "
-        + "AND cc.sessionStartTime >= :start AND cc.sessionStartTime <= :end "
-        + "AND cc.caseNo NOT IN :caseNos")
-    List<CourtCaseEntity> findCourtCasesNotIn(@Param("courtCode") String courtCode,
-        @Param("start") LocalDateTime start,
-        @Param("end") LocalDateTime end,
-        @Param("caseNos") Collection<String> caseNos);
+    @Query(value="select cc.*, grouped_cases.min_created as first_created from court_case cc "
+        + "inner join (select min(created) as min_created, max(created) as max_created, case_no from court_case group_cc "
+        + "where crn = :crn "
+        + "and case_no != :caseNo "
+        + "and session_start_time >= current_date "
+        + "group by case_no, court_code) grouped_cases "
+        + "on cc.case_no = grouped_cases.case_no "
+        + "and cc.created = grouped_cases.max_created",
+        nativeQuery=true)
+    List<CourtCaseEntity> findOtherCurrentCasesByCrn(String crn, String caseNo);
```

Comment:
This doesn't seem to be making any distinction between the 'active' (i.e. the new case with the new probation status) case and the 'other' cases so presumably the 'active' case will be returned in the result set? Not sure if this is filtered down elsewhere but probably worth reflecting this in the method name either way and getting rid of the 'other'

Reply:
☝️ Ignore this I'm being an idiot - of course it's doing the other that's what `and case_no != :caseNo ` is for 🤦

Reply:
Yes - I considered filtering this in the stream.